README.md

```diff
@@ -25,14 +26,21 @@ $ poetry shell
 $ poetry install
 
 # As of 2022/12/06, `origin/fhir-updates` is a currently open development branch. It may be that `main` also has updates that haven't yet made it to release, so you could try running from that branch. If `origin/fhir-updates` is already closed/merged, you can look to see if there are any other open FHIR-related branches https://github.com/INCATools/ontology-access-kit/pulls 
-$ git checkout origin/fhir-updates
+## looks to have been merged?
+##$ git checkout origin/fhir-updates
 
 $ python src/oaklib/cli.py -i OWL_PATH dump -o OUTPATH -O fhirjson --include-all-predicates
+# careful, that's not camel case. It's a lower case j in fhirjson.
 ```
 
 ## Data sources
 Latest releases can be found at the following locations.
-- CompLOINC: TODO
-- go-nucleus: TODO
+- CompLOINC: 
+- RxNorm: 
 - HPO: https://github.com/obophenotype/human-phenotype-ontology/releases/latest/download/hp-full.owl
 - Mondo: https://github.com/monarch-initiative/mondo/releases/latest/download/mondo.owl
+- go-nucleus: 
+
+## Notes
+- RxNorm's ttl file, itself a derivation of the SoT RRF files, needs modified to run here by this script:
```

Comment:
@chrisroederucdenver Thanks, I'll keep this here. I have added this transformation as a step in the `tims-ts` automation as well, so if one tries to convert RxNorm there, no need to know about / run this manually.

https://github.com/HOT-Ecosystem/tims-ts/blob/f98f04205fa3c36125aa0c294286f7a076446943/bin/convert_owl.py#L52-L66